taxi/src/main/java/com/epam/training/taxi/AccountDTOImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.training.taxi;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountDTOImpl implements AccountDTO{
+
+    public static final String PATH_TO_INPUT_CSV = "src/main/resources/accounts.csv";
+    public static final String PATH_TO_OUT_CSV = "./out.csv";
+
+    private List<Account> accounts;
```

Comment:
This field could be a local variable since it is used only in the _readFromCSV(_ method.